### PR TITLE
Replace standard with prettier

### PR DESCRIPTION
**Pull request by jvanbruegge**
Related to #262 

with this the formatter runs on `npm test` (in the root) and on any modified file (via pre-commit hook)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,25 @@
 {
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```
Comment (dmitmel) [inline package.json]: You forgot to add `lint-staged` to dev dependencies

File: package.json [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+    "*.{js,ts}": [
```
Comment (dmitmel) [inline package.json]: There is a file called `packages/sample-app/src/App.tsx`, so the `.tsx` extension should also be added to this list

File: package.json [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
+      "prettier --no-semi --single-quote --write",
+      "git add"
+    ]
+  },
   "scripts": {
-    "test": "lerna run test",
+    "test": "npm run format && lerna run test",
+    "format": "prettier --no-semi --single-quote --write 'packages/**/*.{js,ts}'",
```
Comment (dmitmel) [inline package.json]: Same with extensions as [here](https://github.com/andywer/webpack-blocks/pull/263/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R8)

File: package.json (hunk 1 above)
Comment (dmitmel) [reply package.json]: Ok, this is done

File: package.json [hunk 4]
```diff
@@ -1,12 +1,26 @@
 {
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "prettier --no-semi --single-quote --write",
```
Comment (dmitmel) [inline package.json]: You can configure Prettier via `prettier` key in the `package.json`, so let's move prettier config there to reduce duplication (see [line 15](https://github.com/andywer/webpack-blocks/pull/263/commits/f2dc22756be23685e767f20196e79aaeb8fb9c26#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15))

**Comment by dmitmel** [comment]
Should we format Markdown files with Prettier?

**Comment by andywer** [comment]
Might also make sense to use [`prettier-standard`](https://github.com/sheerun/prettier-standard)?

**Comment by dmitmel** [comment]
@andywer ESLint has more linting options than Standard, but for now we can use Standard.

**Comment by andywer** [comment]
@dmitmel Of course, but I wouldn't change everything at once. Incremental changes are way easier to handle 😉

**Comment by jvanbruegge** [comment]
Changed it to use `prettier-standard`

**Comment by jvanbruegge** [comment]
It looks like prettier-standard does not fix all its issues, which kinda defeats the point of prettier

**Comment by vlad-zhukov** [comment]
Superseded by #275.

**Pull request closed without merging**